en/releases/1.12.md

```diff
@@ -5,26 +5,65 @@
   * Development: [First attempt](https://github.com/PX4/PX4-Autopilot/pull/12072), [Introduction](https://github.com/PX4/PX4-Autopilot/pull/16052), [Improvements](https://github.com/PX4/PX4-Autopilot/pull/16320), [Bugfix zero oscillation](https://github.com/PX4/PX4-Autopilot/pull/16786), [Bugfix position unlock](https://github.com/PX4/PX4-Autopilot/pull/16791), [Bugfix invalid setpoint](https://github.com/PX4/PX4-Autopilot/pull/17078)
 
-* #### Hover thrust independent velocity control gains
-  * Parameters `MPC_{XY/Z}_VEL_{P/I/D}` were replaced with `MPC_{XY/Z}_VEL_{P/I/D}_ACC`, see:<br/>
+* **Hover thrust independent velocity control gains**
+  * Parameters `MPC_{XY/Z}_VEL_{P/I/D}` were replaced with `MPC_{XY/Z}_VEL_{P/I/D}_ACC`, see: 
    [MPC_XY_VEL_P_ACC](../advanced_config/parameter_reference.md#MPC_XY_VEL_P_ACC), [MPC_XY_VEL_I_ACC](../advanced_config/parameter_reference.md#MPC_XY_VEL_I_ACC), [MPC_XY_VEL_D_ACC](../advanced_config/parameter_reference.md#MPC_XY_VEL_D_ACC), [MPC_Z_VEL_P_ACC](../advanced_config/parameter_reference.md#MPC_Z_VEL_P_ACC), [MPC_Z_VEL_I_ACC](../advanced_config/parameter_reference.md#MPC_Z_VEL_I_ACC), [MPC_Z_VEL_D_ACC](../advanced_config/parameter_reference.md#MPC_Z_VEL_D_ACC)
-  * **Attention:** The gains have a new meaning
-    * Scale from velocity error in m/s to acceleration output in m/s^2
-    * Existing gains need to roughly be rescaled by a factor of `* gravitational constant / hover thrust`
-    * Automatic parameter transition assumes 50% hover thrust: ~10m/s^2 / 50% = 20 m/s^2. See [question](https://github.com/PX4/PX4-Autopilot/pull/14823#issuecomment-791357646)
+   
+    :::warning The gains have a new meaning
+    * Scale from velocity error in $m/s$ to acceleration output in $m/s^2$
+    * Existing gains need to roughly be rescaled by a factor of: $gravitational \_ constant / hover\_thrust$
+    * Automatic parameter transition assumes 50% hover thrust: `~10m/s^2 / 50% = 20 m/s^2`.
+      See [question](https://github.com/PX4/PX4-Autopilot/pull/14823#issuecomment-791357646)
+    :::
   * Development: [Logic introduction](https://github.com/PX4/PX4-Autopilot/pull/14749), [Parameter replacement](https://github.com/PX4/PX4-Autopilot/pull/14823)
+* **Improve Rounded Turns ([PR#16376](https://github.com/PX4/PX4-Autopilot/pull/16376))**
+  * Creates a more rounded turn at waypoints in multirotor missions (using L1-style guidance logic in corners)
+
+### VTOL
+
+* **RTL improvements ([PR#16377](https://github.com/PX4/PX4-Autopilot/pull/16377))**
+  * Hardens the RTL safety failsafes taking into consideration the many edge cases when trying to land, depending on the current vehicle mode (Multicopter vs Fixed Wing)
+
+
+### Control
+
+* **Dynamic Notch Filter updated with Gyro FFT ([PR#16385](https://github.com/PX4/PX4-Autopilot/pull/16377))**
+  * Adds dynamic notch filtering to the gyro control data resulting in much smoother control
```

Comment:
This is OK. Is there any relevant information for tuning or whatever? Or is this just magic "goodness"?

Reply:
I pinged a few devs, I will try to add more detail to the release notes, and document properly on the user guides.

https://github.com/PX4/PX4-user_guide/issues/1136